packages/xo-web/src/xo-app/dashboard/overview/index.js

```diff
@@ -141,10 +147,29 @@ class DefaultCard extends Component {
     })
   }
 
+  _canSendTheReport = createSelector(
+    () => this.props.plugins,
+    plugins => {
+      let count = 0
+      forEach(plugins, ({ id, loaded }) => {
+        if (
+          (id === 'usage-report' || id === 'transport-email') &&
+          loaded &&
+          count++ === 2
+        ) {
+          return false
+        }
+      })
+
+      return count === 2
```

Comment:
With a `for` loop, you could test this only once, return directly in the loop and make it clearer.